web/packages/plugins/analysis-platform/src/modules/Asset/components/AssetInfo.tsx

```diff
@@ -1,12 +1,13 @@
 import React, { useContext } from 'react'
 
-import { AuthContext, hasExpertRole } from '@dmt/common'
+import { AuthContext, hasExpertRole } from '@data-modelling-tool/core'
 import { TAssetInfoCardProps } from '../../../Types'
 import { DocumentInfoCard } from '../../../components/DocumentInfoCard'
 import { CreateAnalysisButton } from '../../Analysis'
 import { OnRight } from '../../../components/Design/Styled'
 
 const AssetInfoCardActions = () => {
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore